x/stake/keeper/delegation.go

```diff
@@ -739,6 +739,20 @@ func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress, valAd
 	return ubd, events, nil
 }
 
+func (k Keeper) IsAutoUnDelegate(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) bool {
+	store := ctx.KVStore(k.storeKey)
+	key := GetAutoUnDelegateIndexKey(delAddr, valAddr)
+
+	return store.Has(key)
+}
+
+func (k Keeper) SetAutoUnDelegate(ctx sdk.Context, delAddr sdk.AccAddress, valAddr sdk.ValAddress) {
+	store := ctx.KVStore(k.storeKey)
+
+	key := GetAutoUnDelegateIndexKey(delAddr, valAddr)
+	store.Set(key, []byte{}) // index, store empty bytes
```

Comment:
will this work to store nothing?

Reply:
<img width="1165" alt="image" src="https://github.com/bnb-chain/bnc-cosmos-sdk/assets/25412254/92f77202-8dcc-4e94-a360-c0366143d91b">
there is another case using the syntax